src/AWS.Deploy.CLI/FallbackCredentialsFactoryFactory.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using Amazon.Runtime;
+
+namespace AWS.Deploy.CLI
+{
+    /// <summary>
+    /// Represents a factory for creating AWSCredentials
```

Comment:
Replace explicit string reference of AWSCredentials to a `<see cref="AWSCredentials">`

Reply:
Also, in .NET if the class or members are implementing an interface that has docs and you want the same docs for the implementer, you could use 
```
/// <inheritdoc/>
```